frontend/javascripts/oxalis/model/reducers/settings_reducer.ts

```diff
@@ -178,9 +179,14 @@ function SettingsReducer(state: OxalisState, action: Action): OxalisState {
       const { allowedModes } = state.tracing.restrictions;
 
       if (allowedModes.includes(action.viewMode)) {
-        return updateTemporaryConfig(state, {
+        const newState = updateTemporaryConfig(state, {
           viewMode: action.viewMode,
         });
+        if (action.viewMode !== "orthogonal") {
+          return newState;
+        }
+        // Restore rotation
+        return setRotationReducer(newState, [0, 0, 0]);
```

Comment:
The rotation is only changed if the user does so in flight/oblique mode. Since this affects the matrix (which is also used in orthogonal mode), the rotation needs to be reset. Does this make sense? I also extended the comment a bit.